user/travis-ci-for-private.md

```diff
@@ -23,21 +23,22 @@ the [blog post](https://blog.travis-ci.com/announcing-pull-request-support/) acc
 
 ## Who has access to the builds?
 
-Access rights on Travis CI is based on the access rights on GitHub or Bitbucket:
+Access rights on Travis CI is based on the access rights on GitHub, Bitbucket or GitLab:
 
-- Users that can access a repository on GitHub or Bitbucket can see the build status and logs on Travis CI.
-- Users that can push to a repository on GitHub or Bitbucket can trigger, cancel and restart builds, and change its settings.
-- Users that have admin access to a repository on GitHub or Bitbucket can enable/disable it on Travis CI.
+- Users that can access a repository on GitHub, Bitbucket or GitLab can see the build status and logs on Travis CI.
+- Users that can push to a repository on GitHub, Bitbucket or GitLab can trigger, cancel and restart builds, and change its settings.
+- Users that have admin access to a repository on GitHub, Bitbucket or GitLab can enable/disable it on Travis CI.
 
-To keep the access rights up to date, we sync every user account approximately once every 24 hours with GitHub or Bitbucket. You can use the "Sync account" button on [the profile page](https://travis-ci.com/profile) or `travis sync --com` in the CLI to force a sync.
+To keep the access rights up to date, we sync every user account approximately once every 24 hours with GitHub, Bitbucket or GitLab. You can use the "Sync account" button on [the profile page](https://travis-ci.com/profile) or `travis sync --com` in the CLI to force a sync.
 
```

Comment:
@weban I do not think CLI can be used to force a sync with BB or GL?  So we have to delete this part.

Reply:
Yes CLI is not supported yet